recipes/renet2/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.1" %}
+
+package:
+  name: renet2
+  version: {{ version }}
+
+source:
+  url: https://github.com/sujunhao/RENET2/releases/download/v1.1-alpha/renet2-1.1.tar.gz
```

Comment:
You really want to distribute an alpha release? Those aren't normally meant to be used by anyone.

Reply:
thanks for your suggestion, I will update it to a new version